parsl/executors/workqueue/executor.py

```diff
@@ -407,6 +407,7 @@ def submit(self, func, resource_specification, *args, **kwargs):
         # Create message to put into the message queue
         logger.debug("Placing task {} on message queue".format(task_id))
         category = func.__qualname__ if self.autocategory else 'parsl-default'
+        category = func.__name__ if self.autocategory else 'parsl-default'
```

Comment:
maybe remove the line before this? they are duplicate

Reply:
@ZhuozhaoLi good point, fixed